<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
"http://www.w3.org/TR/html4/loose.dtd">
<!-- saved from url=(0031)http://mrbayes.sourceforge.net/ -->
<html>
<head>
<meta name="generator" content=
"HTML Tidy for Mac OS X (vers 25 March 2009), see www.w3.org">
<meta http-equiv="Content-Type" content=
"text/html; charset=us-ascii">
<title>MrBayes: Home</title>
<link rel="STYLESHEET" type="text/css" href=
"./MrBayes_%20Home_files/mb.css">
<meta name="description" content=
"MrBayes is a free software program which performs Bayesian inference of phylogeny.">
<meta name="keywords" content=
"phylogeny, phylogenies, bayes, systematics, bayesian, mcmc, markov chain monte carlo, metropolis coupled">
</head>
<body bgcolor="#FFFFFF" link="#0000CC" vlink="#0099CC">
<table width="680" border="0" cellpadding="0" cellspacing="20">
<tbody>
<tr>
<td width="100">
<p><img width="100" height="107" border="0" src=
"./MrBayes_%20Home_files/bayes.jpg" alt=
"Photo of Reverend Bayes"></p>
</td>
<td width="580">
<h1>MrBayes: Bayesian Inference of Phylogeny</h1>
</td>
</tr>
<tr>
<td width="100" align="center" valign="top">
<table width="108" border="0" cellpadding="0" cellspacing="0">
<tbody>
<tr>
<td height="30" align="center"><a href=
"http://mrbayes.sourceforge.net/index.php" class=
"selected">Home</a></td>
</tr>
<tr>
<td height="30" align="center"><a href=
"http://mrbayes.sourceforge.net/download.php" class=
"notselected">Download</a></td>
</tr>
<tr>
<td height="30" align="center"><a href=
"http://mrbayes.sourceforge.net/manual.php" class=
"notselected">Manual</a></td>
</tr>
<tr>
<td height="30" align="center"><a href=
"http://mrbayes.sourceforge.net/reportbugs.php" class=
"notselected">Bug Report</a></td>
</tr>
<tr>
<td height="30" align="center"><a href=
"http://mrbayes.sourceforge.net/authors.php" class=
"notselected">Authors</a></td>
</tr>
<tr>
<td height="30" align="center"><a href=
"http://mrbayes.sourceforge.net/links.php" class=
"notselected">Links</a></td>
</tr>
<tr>
<td height="115" align="center"><a href=
"http://www.vr.se/"><img src="./MrBayes_%20Home_files/vr.png" alt=
"Vetenskapsradet" width="108" height="112"></a></td>
</tr>
<tr>
<td height="90" align="center"><a href=
"http://www.nih.gov/"><img src="./MrBayes_%20Home_files/nih.gif"
alt="National Institutes of Health" width="108" height=
"78"></a></td>
</tr>
<tr>
<td height="90" align="center"><a href=
"http://www.nsf.gov/"><img src="./MrBayes_%20Home_files/nsf.gif"
alt="National Science Foundation" width="90" height="80"></a></td>
</tr>
</tbody>
</table>
</td>
<td align="left" valign="top">
<hr width="100%">
<table width="100%" border="0" cellpadding="0" cellspacing="0">
<tbody>
<tr>
<td align="left" valign="top">
<p>MrBayes is a program for Bayesian inference and model choice
across a wide range of phylogenetic and evolutionary models.
MrBayes uses Markov chain Monte Carlo (MCMC) methods to estimate
the posterior distribution of model parameters.</p>
<p>Program features include:</p>
<ul>
<li>A common command-line interface across Macintosh, Windows, and
UNIX operating systems;</li>
<li>Extensive help available from the command line;</li>
<li>Analysis of nucleotide, amino acid, restriction site, and
morphological data;</li>
<li>Mixing of data types, such as molecular and morphological
characters, in a single analysis;</li>
<li>Easy linking and unlinking of parameters across data
partitions;</li>
<li>An abundance of evolutionary models, including 4x4, doublet,
and codon models for nucleotide data and many of the standard rate
matrices for amino acid data;</li>
<li>Estimation of positively selected sites in a fully hierarchical
Bayesian framework;</li>
<li>Full integration of the BEST algorithms for the multi-species
coalescent.</li>
<li>Support for complex combinations of positive, negative, and
backbone constraints on topologies;</li>
<li>Model jumping across the GTR model space and across fixed rate
matrices for amino acid data;</li>
<li>Monitoring of convergence during the analysis, and access to a
wide range of convergence diagnostics tools after the analysis has
finished;</li>
<li>Rich summaries of posterior samples of branch and node
parameters printed to majority rule consensus trees in FigTree
format;</li>
<li>Implementation of the stepping-stone method for accurate
estimation of model likelihoods for Bayesian model choice using
Bayes factors;</li>
<li>The ability to spread jobs over a cluster of computers using
MPI (for Macintosh (OS X) and UNIX environments only);</li>
<li>Support for the BEAGLE library, resulting in dramatic speedups
for codon and amino acid models on compatible hardware (NVIDIA
graphics cards);</li>
<li>Checkpointing across all models, allowing the user to
seemlessly extend a previous analysis or recover from a system
crash;</li>
</ul>
</td>
</tr>
</tbody>
</table>
</td>
</tr>
</tbody>
</table>
<span id="buffer-extension-hover-button" style=
"display: none;position: absolute;z-index: 8675309;width: 100px;height: 25px;background-image: url(chrome-extension://noojglkidnpfjbincgijbaiedldjfbhh/data/shared/img/buffer-hover-icon@1x.png);background-size: 100px 25px;opacity: 0.9;cursor: pointer;"></span>
</body>
</html>
